### PR TITLE
Log elements can be created by function calls. & Some Refactor

### PR DESCRIPTION
**Pull request by tomorrow9913**
### Description

- Initialize basic elements with function calls to unify logs
- Removed unnecessary code for now
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (code improvement that does not add functionality)
- [ ] Documentation update (README, changelog, or other documentation)
- [ ] Other (please explain):

### Checklist

- [x] I have read the [CONTRIBUTING.md](./.github/docs/CONTRIBUTING.md) document
- [x] My code follows the code style of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings or errors

> [!WARNING]
> There may be merge conflicts due to minor code modifications.

### CODE REVIEW

**Comment by tomorrow9913** [comment]
약간의 코드 수정이 있어 작업중인 브랜치에 당겨갈 경우 약간의 충돌이 발생 할 수 있음.

추후 이 브랜치를 통해 map 설정 관련 작업을 추가할 에정

File: src/lsm/enforcement.bpf.c
Comment (KoYejune0302) [inline src/lsm/enforcement.bpf.c:1]: header 순서 불편하네요

**Review by KoYejune0302 — APPROVED** [review]
LGTM!

**Comment by tomorrow9913** [comment]
수정후 빌드가 안되네요 해결바랍니다

**Pull request merged**